lib/play.rb

```diff
@@ -1,36 +1,35 @@
 require_relative './commands'
 require_relative './code'
-require_relative './response'
+require_relative './compare_code'
 require_relative './messages'
+require_relative './get_input'
 
 class Play
   include Commands, Messages
-   MAX_GUESS_COUNT ||= 10
+   MAX_TURN_COUNT ||= 10
```

Comment:
I don't think there is much benefit to memoizing a constant because it doesn't change once it is loaded into memory.

Reply:
I ended up memoizing because it was returning a NameError "uninitialized constant Play::MAX_TURN_COUNT". Would it be better to use rescue instead to get around the error?